### PR TITLE
Fix monitoring email when too many tasks have failed

### PR DESCRIPTION
**Pull request by juspence**
@RedHatProductSecurity/corgi-devs Quick review please. I've tested locally and this seems to work, but I could also deploy to stage (without merging) if people prefer.

### CODE REVIEW

File: corgi/tasks/monitoring.py [hunk 1]
```diff
@@ -83,29 +84,41 @@ def email_failed_tasks():
...
+    failed_tasks = TaskResult.objects.filter(
+        status__exact="FAILURE",
+        date_done__gte=failed_tasks_threshold,
+    ).using("read_only")
 
+    failed_tasks_count = failed_tasks.count()
     subject = (
-        f"Failed Corgi Celery tasks after "
-        f"{failed_tasks_threshold.date()}: {failed_tasks.count()}"
+        f"Failed Corgi Celery tasks after " f"{failed_tasks_threshold.date()}: {failed_tasks_count}"
```
Comment (mprpic) [inline corgi/tasks/monitoring.py]: Black mangled this a bit, jfyi.

File: corgi/tasks/monitoring.py [hunk 2]
```diff
@@ -83,29 +84,41 @@ def email_failed_tasks():
...
     )
 
-    failed_tasks = "\n".join(
-        f"{task_name}: args={task_args}, kwargs={task_kwargs}\n"
-        f"result={result}\n"
-        f"{traceback}\n"
-        for (task_name, task_args, task_kwargs, result, traceback) in failed_tasks.iterator(
-            chunk_size=500
+    if failed_tasks_count < 5000:
```
Comment (mprpic) [inline corgi/tasks/monitoring.py:97]: 5000 still seems like a lot, will that not go over the email message size limit?

**Review by mprpic — APPROVED** [review]
Looks good!

File: corgi/tasks/monitoring.py (hunk 2 above)
Comment (juspence) [reply corgi/tasks/monitoring.py:97]: It depends on the message size (length of args + kwargs + traceback). The largest email I've seen came in with 17k errors, but I could lower to 2k or 1k if you think that's safer.

Comment (mprpic) [reply corgi/tasks/monitoring.py:97]: Ha, if 17k worked then I guess 5k is fine :)

**Pull request merged**